src/gg.regression.unity.bots/Runtime/Scripts/StateRecorder/BotSegments/Models/BotSequence.cs

```diff
@@ -35,6 +35,13 @@ public class BotSequence
          */
         public string description;
 
+        /**
+         * Not persisted or written to json, populated on load
+         */
+        public string resourcePath;
+
+        public static BotSequence ActiveBotSequence = null;
```

Comment:
The changes in these areas get a big ugly, but I tried to minimize the 'full re-write' that could have happened here.

We have code now to manage each of these possible playback cases and their 'status' on each entity in their chain as much as possible...

- Remote Work . which runs a botsequence , which uses the playback controller to run its segments
- Botsequence , which uses the playback controller to run its segments
- Zip, which uses the playback controller to run its segments
- Single Segment, which uses the playback controller to run its segment

So...
- the playbackcontroller can tell us if there is are active segments, but can't tell us if they are in a sequence
- a botsequence can now tell us if there is an active sequence, so we can give more details than just true/false segments are running
- a work assignment, can tell us its details, and also checks for sequence/segments active